src/main/java/uk/gov/hmcts/reform/coh/controller/decision/DecisionResponse.java

```diff
@@ -5,10 +5,11 @@
 import uk.gov.hmcts.reform.coh.controller.decisionreplies.DecisionReplyResponse;
 import uk.gov.hmcts.reform.coh.controller.state.StateResponse;
 
+import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.List;
 
-public class DecisionResponse {
+public class DecisionResponse implements Serializable {
```

Comment:
Why Serializable?

Reply:
Jackson wants serializable objects ? Also sonarqube underlined it in red and told me to make it serializable

Reply:
Hmm. Jackson already serialises it 🤷‍♂️ Okay. Just ignore this comment